code/game/objects/items/storage/toolbox.dm

```diff
@@ -299,3 +299,129 @@
 /obj/item/storage/toolbox/haunted
 	name = "old toolbox"
 	custom_materials = list(/datum/material/hauntium = 500)
+
+/obj/item/storage/toolbox/captain
+	name = "\proper captain's toolbox"
```

Comment:
The `\proper` here seems weird to me, is that common?

"You throw captain's toolbox" reads clunkier to me than "You throw the captain's toolbox"

Reply:
I copied from the Command lockers because I assumed it was the second option already.
I can change it in a sec.

Reply:
Ergh, those should be changed too x_x

Don't care about in this PR but Yeah